test_files/clutz.no_externs/strip_clutz_type.js

```diff
@@ -1,4 +1,5 @@
 goog.module('test_files.clutz.no_externs.strip_clutz_type');var module = module || {id: 'test_files/clutz.no_externs/strip_clutz_type.js'};
+Object.defineProperty(exports, "__esModule", { value: true });
```

Comment:
Interesting, we did have some of those before, right? I'll verify this change through global presubmit before merging.